ingress/loadbalancer.go

```diff
@@ -2,11 +2,31 @@ package ingress
 
```

Comment:
I think all nginx specific stuff should go into its own `nginx.go` file. Then if we add other load balancers, it's just a new file like `brocade.go`. The api interfaces should remain in here. What do you think?